src/pybind/mgr/orchestrator/_interface.py

```diff
@@ -1382,6 +1382,8 @@ def to_json(self):
         out['version'] = self.version
         out['status'] = self.status
         out['status_desc'] = self.status_desc
+        if self.daemon_type == 'osd':
+            out['osdspec_affinity'] = self.osdspec_affinity
```

Comment:
also from_json?

Reply:
the `from_json` should be handled already